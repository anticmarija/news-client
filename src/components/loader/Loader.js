import React from "react";
import * as loaderGif from "../../assets/loader.gif";
import { LoaderWrapperStyle } from "./Loader.style";

const Loader = () => {
  return (
    <LoaderWrapperStyle>
      <img alt="loader" src={loaderGif} />
    </LoaderWrapperStyle>
  );
};

export default Loader;
