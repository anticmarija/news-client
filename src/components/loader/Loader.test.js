import React from "react";
import Loader from "./Loader";
import { render } from "@testing-library/react";

describe("Test Loader component", () => {
  it("should render correctly", () => {
    const { container } = render(<Loader />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
