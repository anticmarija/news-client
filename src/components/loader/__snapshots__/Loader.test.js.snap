// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Loader component should render correctly 1`] = `
<div
  class="Loaderstyle__LoaderWrapperStyle-sc-18hrjj-0 fhBkiV"
>
  <img
    alt="loader"
    src="[object Object]"
  />
</div>
`;
